src/main/java/com/example/Storehouse/logger/Print.java

```diff
@@ -0,0 +1,28 @@
+package com.example.Storehouse.logger;
+
+import com.example.Storehouse.controllers.StorehouseController;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+@Component
+public class Print implements iPrint{
```

Comment:
typically this service is called LoggerService so please rename

Reply:
done